pay-api/src/pay_api/services/eft_service.py

```diff
@@ -24,3 +28,8 @@ class EftService(DepositService):
     def get_payment_method_code(self):
         """Return EFT as the system code."""
         return PaymentMethod.EFT.value
+
+    def create_invoice(self, payment_account: PaymentAccount, line_items: [PaymentLineItem], invoice: Invoice,
```

Comment:
The base deposit service class already has an implementation of the required methods. I added create_invoice with a comment so it is clear that the Invoice references will be created later via the TDI17 processing.